src/EnergyPlus/HeatingCoils.cc

```diff
@@ -3320,6 +3344,73 @@ namespace HeatingCoils {
 
 	}
 
+	void
+	SetHeatingCoilData(
+		int const CoilNum, // Number of electric or gas heating Coil
+		bool & ErrorsFound, // Set to true if certain errors found
+		Optional_bool DesiccantRegenerationCoil, // Flag that this coil is used as regeneration air heating coil
```

Comment:
I suggested this method to "set" data as opposed to the mining method, "get" data, currently used. This function is analogous to SetDXCoolingCoilData so is a better approach than using multiple or complex (multiple variables) mining functions. We did discuss several options for getting rid of optional arguments for these "set" functions.